src/modality-specific-files/physiological-and-other-continuous-recordings.md

```diff
@@ -29,29 +29,118 @@ sub-<label>/[ses-<label>/]
+    },
+  },
+  "task-nback_stim.json": "",
+  "task-nback_stim.tsv.gz": "",
+}) }}
+
+For multi-echo data, a single `_<physio|stim>.<tsv.gz|json>` file without the
+[`echo-<index>`](../appendices/entities.md#echo) entity applies to all echos of
+a particular run.
+For example:
 
+<!-- This block generates a file tree.
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_filetree_example({
+  "sub-01": {
+    "func": {
+      "sub-01_task-nback_run-1_echo-1_bold.nii.gz": "",
+      "sub-01_task-nback_run-1_echo-2_bold.nii.gz": "",
+      "sub-01_task-nback_run-1_echo-3_bold.nii.gz": "",
+      "sub-01_task-nback_run-1_physio.tsv.gz": "",
+      "sub-01_task-nback_run-1_stim.tsv.gz": "",
+    },
+  },
+}) }}
+
+**Storing different recordings**.
 The [`recording-<label>`](../appendices/entities.md#recording)
 entity MAY be used to distinguish between several recording files.
-For example `sub-01_task-bart_recording-eyetracking_physio.tsv.gz` to contain
-the eyetracking data in a certain sampling frequency, and
-`sub-01_task-bart_recording-breathing_physio.tsv.gz` to contain respiratory
-measurements in a different sampling frequency.
-
-Physiological recordings (including eyetracking) SHOULD use the `_physio`
-suffix, and signals related to the stimulus SHOULD use `_stim` suffix.
+Recordings with different metadata such as sampling frequencies
+or recording device MUST be stored in separate files with different
+[`recording-<label>`](../appendices/entities.md#recording) entities.
+For example, given a multi-echo acquisition corresponding to a breath-holding
+task (`task-bht`) for which pulse and respiratory movement were
+sampled at different frequencies, recordings are separated as follows:
 
-The following table specifies metadata fields for the `*_<physio|stim>.json` file.
+<!-- This block generates a file tree.
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_filetree_example({
+  "sub-01": {
+    "func": {
+      "sub-01_task-bht_echo-1_bold.nii.gz": "",
+      "sub-01_task-bht_echo-2_bold.nii.gz": "",
+      "sub-01_task-bht_echo-3_bold.nii.gz": "",
+      "sub-01_task-bht_recording-cardiac_physio.json": "",
+      "sub-01_task-bht_recording-cardiac_physio.tsv.gz": "",
+      "sub-01_task-bht_recording-respiratory_physio.json": "",
+      "sub-01_task-bht_recording-respiratory_physio.tsv.gz": "",
+    },
+  },
+}) }}
+
+**Example datasets**.
+[Example datasets](https://bids-standard.github.io/bids-examples/#dataset-index)
+with physiological data have been formatted using this specification
+and can be used for practical guidance when curating a new dataset:
+
+-   [`7t_trt`](https://github.com/bids-standard/bids-examples/tree/master/7t_trt)
+-   [`ds210`](https://github.com/bids-standard/bids-examples/tree/master/ds210)
```

Comment:
After fixing a merged conflict (the spec can now use mkdocs admonition) I realize that this is now duplicated. 
@oesteban : for most datatype pages we try to keep the link to the examples very close to the top. Not a hard rule, just an FYI. 😉

Reply:
Resolved, and applied the same pattern (admonition, at the top) to the new eyetrack examples

Reply:
Thanks!!